src/mc-rangeopts.c

```diff
@@ -191,7 +191,7 @@ bool mc_RangeOpts_parse(mc_RangeOpts_t *ro, const bson_t *in, bool use_range_v2,
     if (ro->trimFactor.set) {
         if (!use_range_v2) {
             // Once `use_range_v2` is default true, this block may be removed.
-            CLIENT_ERR("trimFactor is not supported for QE range v1");
+            CLIENT_ERR("%strimFactor is not supported for QE range v1", error_prefix);
```

Comment:
Change it to be `%s: ...` for readability unless you can guarantee `error_prefix` always has a trailing space.

Reply:
Kinda got fed up with the inconsistencies in error prefixing and just added a new macro to do it, let me know what you think of the approach. Changed it in this file and fle2-encryption-placeholder.